src/modules/core/helpers/utils.helper.ts

```diff
@@ -36,4 +35,11 @@ export class UtilsHelper {
     return value;
   }
 
+  static indefiniteArticle(word: string): string {
```

Comment:
btw I don't mean for you to change this, it's a good helper ;) it might induce errors in the future but people must take it for what it is, a helper ;)

Reply:
I was looking into the same, we should probably use a third party library that takes into consideration most of the cases (when the need arises). But for now it looks great 😄